### DISCUSSION TITLE
mvn clean install fails for knowledge-platform repo

### DISCUSSION THREAD

**Discussion by archit15singh** [Sunbird-Knowlg/Community#163] [Q&A] [answered]
macOS

for the repo: https://github.com/Sunbird-Knowlg/knowledge-platform
i ran this: sh ./local-setup.sh (success, all services are up and running)
and when i run: mvn -e clean install -DskipTests as mentioned [here](https://github.com/Sunbird-Knowlg/knowledge-platform#running-content-service)

i get this error:
[log.log](https://github.com/Sunbird-Knowlg/Community/files/13226531/log.log)

<img width="1728" alt="Screenshot 2023-11-01 at 5 18 47 PM" src="https://github.com/Sunbird-Knowlg/Community/assets/131599737/89ddb107-42af-4cf0-a433-408f3fd6d4e4">

**Comment by aimansharief** [comment]
@archit15singh can you please mention which branch you are using?

**Comment by archit15singh** [comment]
@aimansharief master

**Reply by aimansharief** [reply, marked as answer]
@archit15singh please try to build with this command 
`mvn clean install -DskipTests -DCLOUD_STORE_GROUP_ID=org.sunbird -DCLOUD_STORE_ARTIFACT_ID=cloud-store-sdk_2.12 -DCLOUD_STORE_VERSION=1.4.6`

**Comment by archit15singh** [comment]
thank you! @aimansharief this worked! 
updating the README.md to reflect this. will issue a PR

**Reply by pallakartheekreddy** [reply]
@archit15singh We would greatly appreciate your assistance in resolving this documentation issue. Thank you for your willingness to contribute!

**Reply by pallakartheekreddy** [reply]
@archit15singh Hope your query is now resolved. Please mark the thread as answered.

**Comment by archit15singh** [comment]
https://github.com/Sunbird-Knowlg/knowledge-platform/pull/1006
@aimansharief 
the jira github checks are failing

**Reply by pallakartheekreddy** [reply]
@archit15singh Yes, as part of the process, each pull request (PR) should be associated with the respective JIRA ticket. It seems that this check is failing.

**Comment by archit15singh** [comment]
so should i create a JIRA ticket for this? @pallakartheekreddy
